src/Hazelcast.Net.Testing/Hazelcast.Net.Testing.csproj

```diff
@@ -21,14 +17,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ApacheThrift" Version="0.13.0.1" />
+    <PackageReference Include="ApacheThrift" Version="0.15.0" />
```

Comment:
@zpqrtbnk A question: Did you used 0.15.0 version of thrift compiler in this pr to generate remote controller bindings?